FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Scheduler/Jobs/AzureBlobJobStore.cs

```diff
@@ -141,8 +141,7 @@ public async Task<bool> CompleteJobAsync(Job job, CancellationToken cancellation
         {
```

Comment:
Nit: Can we change this interface **CompleteJobAsync** to **CommitJobAsync** or something else? Because now we will save completedJob to **completed** blobs and falied job to **failed** blobs.

Reply:
Thanks. We'll change the name later.